src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs

```diff
@@ -716,6 +719,8 @@ private void OnUpdateGraphCompleted(AsyncTask task)
                 if (node == null)
                     continue;
 
+                nodesWithInfos.Add(guid);
```

Comment:
How was it working before without adding this guid here?

Reply:
It was working before I broke it by adding the boolean `BlockInfoBubbleUpdates`
```
                using (Disposable.Create(() => { node.BlockInfoBubbleUpdates = true; }, () => { node.BlockInfoBubbleUpdates = false; }))
                {
                    node.Info(string.Join(Environment.NewLine, info.Value.Select(w => w.Message)));
                }
```
All calls to ` node.Info` used to trigger an UpdateBubbleInfo directly. But with the `BlockInfoBubbleUpdates` we stop UpdateBubbleInfo
```
        public void UpdateBubbleContent()
        {
            if (NodeModel.BlockInfoBubbleUpdates)
            {
                return;
            }

```

Reply:
In this PR, by adding the node guids to `nodesWithInfos` we ensure that all affected nodes will call UpdateBubbleContent only one time at the end of the graph execution